tpm2/encoding_test.go

```diff
@@ -312,7 +312,7 @@ func TestEncodeAuthArea(t *testing.T) {
 		t.Fatalf("got: %v, want: %v", pwAuth, want)
 	}
 
-	pwAuth, err = encodeAuthArea(HandlePasswordSession, "")
+	pwAuth, err = encodeAuthArea(AuthCommand{Session: HandlePasswordSession, Attributes: AttrContinueSession, Auth: []byte("")})
```

Comment:
`[]byte("")` is `EmptyAuth` in the spec, right? Should we make a var for it in constants.go?